libs/remix-ui/static-analyser/src/lib/remix-ui-static-analyser.tsx

```diff
@@ -469,14 +515,24 @@ export const RemixUiStaticAnalyser = (props: RemixUiStaticAnalyserProps) => {
           {state.file}
         </span>
       </div>
-      <br/>
       {Object.entries(warningState).length > 0 &&
         <div id='staticanalysisresult' >
+          <RemixUiCheckbox
+          id="showLibWarnings"
+          name="showLibWarnings"
+          categoryId="showLibWarnings"
+          inputType="checkbox"
+          checked={showLibsWarning}
+          label="Show library files analysis"
```

Comment:
The rewording needs to be be reviewed. this should be something like:
`Display all results` and a title should give more information like `when checked, all the results are displayed, including report coming from external contract libraries`.
and this should be unchecked by default

Reply:
done